app/DoctrineMigrations/Version20200428072628.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Wallabag\CoreBundle\Doctrine\WallabagMigration;
+
+/**
+ * Renamed Piwik to Matomo in configuration 
+ */
+final class Version20200428072628 extends WallabagMigration
+{
+    public function up(Schema $schema) : void
+    {
+        $this->addSql('UPDATE ' . $this->getTable('internal_setting', true) . ' SET name = "matomo_enabled" where name = "piwik_enabled";');
```

Comment:
On PostgreSQL you can't use `"` to enclose values as it's restricted to column and table names. You must change `"` to `'`, see failing tests.